index.html

```diff
@@ -0,0 +1,218 @@
+            </a>
+          </li>
+          <li class="top_toolbar_items">
+            <a href="#">English</a>
+          </li>
+          <li class="top_toolbar_items">
+            <a href="#">My Page</a>
+          </li>
+          <li class="top_toolbar_items">
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+      </div>
+
+      <nav>
+        <div class="nav_container">
+          <a href="#home" id="brand">
+            <img class="header_logo" src="images/nutrition-logo.png" alt="brand logo">
+          </a>
+          <ul class="desktop desktop-nav">
+            <li class="nav-links">
+              <a href="#program_section">Programs</a>
+            </li>
+            <li class="nav-links">
+              <a href="#speakers_section">Speakers</a>
+            </li>
+            <li class="nav-links">
+              <a href="#partners">Sponsor</a>
+            </li>
+            <li class="nav-links">
+              <a href="about.html">About</a>
+            </li>
+              <li class="nav-links campaign_link">
+                <a href="#contact">Campaign</a>
+              </li>
+          </ul>
+          <div class="mobile-menu-bars mobile" id="menu-bars">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </div>
+        </div>
+      </nav>
+
+      <nav class="overlay overlay-slide-left" id="overlay">
+        <ul class="mobile-nav-items">
+          <li id="nav-1"><a href="#home" class="nav-mobile-links">Home</a></li>
+        <li id="nav-1"><a href="#program_section" class="nav-mobile-links">Programs</a></li>
+        <li id="nav-2"><a href="#speakers_section" class="nav-mobile-links">Speakers</a></li>
+        <li id="nav-3"><a href="#partners" class="nav-mobile-links">Sponsor</a></li>
+        <li id="nav-3"><a href="about.html" class="nav-mobile-links">About</a></li>
+        <li id="nav-3"><a href="#" class="nav-mobile-links">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+ 
+    <section class="header_section" id="home">
+      
+      <div class="header_section_main_div">
+        <h2 class="stem_by_us"> Living Hapily with Diabetes</h2>
+
+        <div class="header_text_div">
+          <p>AFRICAN DIABETES ASSOCIATION 2022!</p>
+        </div>
+
+        <p class="header_about_stem">
+          It is with great pleasure that we invite you to the 
+          2nd edition of diabetes Conference. This conference is an
+          event hosted by African top health experts. In 2022, it will 
+          be stream live on all Africa Diabetes Association media pages. 
+          The theme this 2022 is “CHANGING THE MIND“.
+        </p>
+        <h2 class="date_of_event">
+          2022.11.25(FRI) ~ 26(SAT)
+        </h2>
+        <h3 class="event_address">
+          @Hilton hotel confernce hall yaounde,Cameroon.
+        </h3>
+      </div>
+    </section>
```

Comment:
- Please note that there are some discrepancies between the home page for your site and the home page in the design template. Please try to fix this and try to replicate the design template as closely as possible, and pay keen attention to the fonts, font size, colors, margins e.t.c 🙏  🙏. Replicating exact design specs give confidence to recruiters in your ability to build exact design templates 😄 😄 
Your project (Pay attention to the font size of your title header 😄):
![home project student](https://user-images.githubusercontent.com/73607512/192114411-48214389-e34c-4240-86de-c953832ee4ad.PNG)

Design template:
![home project figma](https://user-images.githubusercontent.com/73607512/192114421-efe45ee6-a5bc-42fe-ad6a-ddb556092b52.PNG)